builder/builder.mk

```diff
@@ -147,6 +147,9 @@ INGRESS_TEST_LOCAL_ADMIN_PORT = 8877
 INGRESS_TEST_MANIF_DIR = $(BUILDER_HOME)/../manifests/emissary/
 INGRESS_TEST_MANIFS = emissary-crds.yaml emissary-emissaryns.yaml
 
+VERSION ?= $(shell go run $(OSS_HOME)/tools/src/git-godescribe)
+IS_DIRTY=$(shell git status --porcelain)
+
```

Comment:
This didn't actually need to move up in the file, but it's probably a readability win to do so.

If the tree is dirty, `git-godescribe` includes a timestamp.  Because of that, we kinda want to only call it once.

So `VERSION := $(or $(VERSION),$(shell go run $(OSS_HOME)/tools/src/git-godescribe))` would probably be better (Make variables are usually lazy-evaluated, `:=` makes them be eager).